tests/Doctrine/Tests/Common/Reflection/AnnotationClassWithScopeResolution.php

```diff
@@ -9,13 +9,13 @@
  *   key = "value"
  * )
  */
-class AnnotationClassWithScopeResolution {
+class AnnotationClassWithScopeResolution
+{
 
-  const foo = \stdClass::class;
+    const FOO = \stdClass::class;
```

Comment:
Missing visibility, maybe Doctrine CS should enforce it for 7.1+?

Reply:
That sounds like an awesome addition, since we have PHP 7.1+ on our packages I think we should enforce people to make things explicit. We should find/build the sniff to do that though.

Reply:
Have a look at [slevomat/coding-standard](https://github.com/slevomat/coding-standard), they provide a bunch of very interesting 7.1+ sniffs; I'm happily using them elsewhere. 👍

Reply:
Seems neat, could you send a PR to `doctrine/coding-standard`?